src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+
+        <label name="surname">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) fix code style all over the page